hub/api/tests/test_update_samples.py

```diff
@@ -216,6 +238,44 @@ def test_failures(memory_ds):
     assert memory_ds.labels.shape == (10, 1)
 
 
+def test_subslice_failure(memory_ds):
+    memory_ds.create_tensor("tensor")
+    memory_ds.tensor.extend(np.ones((3, 28, 28, 3)))
+
+    # cannot update multiple sample subslices at the same time
```

Comment:
dynamic tensors make this weird, i would rather avoid edge case code for right now

Reply:
Yup, I'm fine with this not working. I don't see a lot of use cases for updating multiple samples in a single go.